.github/workflows/tp.integration.tests.yml

```diff
@@ -107,6 +109,9 @@ jobs:
       - name: Initialize Traffic Ops Database
         id: todb
         uses: ./.github/actions/todb-init
+      - name: Initialize Traffic Vault Database
+        id: tvdb
+        uses: ./.github/actions/tvdb-init
```

Comment:
Could we use the same PostgreSQL instance as the TODB but just add another database? Seems like that might be faster.

Reply:
That is what this does, it logs in using `traffic_ops` and creates `traffic_vault` user and db

Reply:
Yeah, I see that now. My bad.